common/download.go

```diff
@@ -122,10 +125,51 @@ func (d *DownloadClient) Get() (string, error) {
 		finalPath = url.Path
 		log.Printf("[DEBUG] Using local file: %s", finalPath)
 
-		// Remove forward slash on absolute Windows file URLs before processing
-		if runtime.GOOS == "windows" && len(finalPath) > 0 && finalPath[0] == '/' {
-			finalPath = finalPath[1:len(finalPath)]
+		// FIXME:
+		//	cwd should point to the path relative to client.json, but
+		//	since this isn't exposed to us anywhere, we use os.Getwd()
+		//	to figure it out.
+		cwd,err := os.Getwd()
+		if err != nil {
+			return "", fmt.Errorf("Unable to get working directory")
 		}
+
+		// convert the actual file uri to a windowsy path
+		// (this logic must correspond to the same logic in common/config.go)
+		if runtime.GOOS == "windows" {
```

Comment:
This block is getting pretty long. Can we move it into a separate function so it can be tested separately?